Expand all appearances of uint to unsigned int

arkamar opened the pull request:
This commit resolves a build issue with musl libc found in [Gentoo](https://bugs.gentoo.org/712624), where `uint` definition is not available in two places and simplest solution is to expand it to `unsigned int`.

arkamar committed:
Use unsigned int where uint is not defined

This commit resolves a build issue with musl libc found in Gentoo [1],
where uint definition is not available in two places and simplest
solution is to expand it to unsigned int.

[1] https://bugs.gentoo.org/712624

Signed-off-by: Petr Vaněk <arkamar@atlas.cz>

jktjkt reviewed (changes requested):
In #22 the maintainer has requested a different approach.

jktjkt reviewed (changes requested):
In https://github.com/tat/mimetic/pull/22 the maintainer requested a different approach.

arkamar committed:
Expand all appearances of uint to unsigned int

This commit changes all uint to unsigned int as it was suggested in [1].

[1] https://github.com/tat/mimetic/pull/22#issuecomment-817549157

Signed-off-by: Petr Vaněk <arkamar@atlas.cz>

arkamar commented:
I decided to go with second option, I have changed it everywhere.
